sympy/core/tests/test_args.py

```diff
@@ -2014,6 +2014,11 @@ def test_sympy__functions__combinatorial__numbers__carmichael():
     assert _test_args(carmichael(x))
 
 
+def test_sympy__functions__combinatorial__numbers__mobius():
+    from sympy.functions.combinatorial.numbers import mobius
+    assert _test_args(mobius(2))
```

Comment:
The other test like this that imports from ntheory should be removed. Currently it gives a deprecation warning when running the core tests:
```
=============================== warnings summary ===============================
sympy/core/tests/test_args.py::test_sympy__ntheory__residue_ntheory__mobius
  /home/runner/work/sympy/sympy/sympy/core/tests/test_args.py:5014: SymPyDeprecationWarning: 
  
  The `sympy.ntheory.residue_ntheory.mobius` has been moved to `sympy.functions.combinatorial.numbers.mobius`.
  
  See https://docs.sympy.org/latest/explanation/active-deprecations.html#deprecated-ntheory-symbolic-functions
  for details.
  
  This has been deprecated since SymPy version 1.13. It
  will be removed in a future version of SymPy.
  
    assert _test_args(mobius(2))
```